BUILD

```diff
@@ -0,0 +1,30 @@
+genrule(
+  name = "next",
+  srcs = glob([
+    "_speakers/**",
+    "components/**",
+    "data/**",
+    "Dockerfile",
+    "firebase.json",
+    "next.config.js",
+    "package-lock.json",
+    "package.json",
+    "pages/**",
+    "README.md",
+    "static/**",
+    "stories/**",
+    "utils/**",
+    "WORKSPACE"
+  ]) + ["//_posts:copy"],
+  cmd = "tar -czh . | docker build -t cphjs - && docker run --rm -v $$PWD/$$(dirname $(location build.tar.gz)):/app/mountedout cphjs bash -c 'tar -C out -czf mountedout/build.tar.gz .'",
```

Comment:
This looks crazy but what it does:
- uses tar to send all files to docker: that is because symlinks in bazel sandbox
- docker build the docker file
- run the docker image to extract the build out
- mount the volume where bazel expect the tar file to be outputtet
- tar the output of next.js